### PR TITLE
Add a test to make sure lldb/sos is functional

### PR DESCRIPTION
**Pull request by omajid**
Please note that some of the tests are disabled because they fail on the builds I am running it against and I dont want to block future builds where things don't get any worse.

### CODE REVIEW

**Comment by omajid** [comment]
cc @davemulford @RheaAyase

File: debugging-sos-lldb-via-core/test.sh [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+lldb-core 'clrthreads' > lldb.out
+cat lldb.out
+thread_id=$(grep -A5 'ID OSID ThreadOBJ' lldb.out | tail -4 | grep -v 'Finalizer' | head -1 | awk '{print $1}')
+lldb-core "thread select ${thread_id}" 'clrstack' > lldb.out
+cat lldb.out
+ip=$(grep 'OS Thread Id:' lldb.out -A5 | tail -n3 | grep -v -F '[Prestub' | head -1 | cut -d' ' -f2)
+lldb-core "ip2md ${ip}" > lldb.out
+cat lldb.out
+if ! grep 'IsJitted' lldb.out ; then
+    echo 'sJitted field not found in ip2md'
```
Comment (aslicerh) [inline debugging-sos-lldb-via-core/test.sh]: nit - ctypo: IsJitted?

File: debugging-sos-lldb-via-core/test.sh [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+lldb-core 'clrstack' > lldb.out
+cat lldb.out
+stack_pointer=$(grep 'OS Thread Id' lldb.out -A3 | tail -1 | awk '{ print $1}')
+lldb-core "dumpstack ${stack_pointer}"
+cat lldb.out
+
+echo "[eestack]"
+lldb-core 'eestack' > lldb.out
+cat lldb.out
+grep 'Child-SP         RetAddr          Caller, Callee' lldb.out
```
Comment (aslicerh) [inline debugging-sos-lldb-via-core/test.sh]: nit: Is the whitespace here going to stay consistent? Should something like \s\+ be used?

File: debugging-sos-lldb-via-core/test.sh (hunk 1 above)
Comment (omajid) [reply debugging-sos-lldb-via-core/test.sh]: Thanks! Fixed.

File: debugging-sos-lldb-via-core/test.sh (hunk 2 above)
Comment (omajid) [reply debugging-sos-lldb-via-core/test.sh]: Thanks! Fixed.

**Comment by RheaAyase** [comment]
Thanks y'all! I guess that we can merge and run this thing.

**Pull request merged**